README.md

```diff
@@ -1,29 +1,29 @@
-[![Angular GitHub CI/CD](https://github.com/Nikemaks/DevNoteProximity/actions/workflows/main.yaml/badge.svg)](https://github.com/Nikemaks/DevNoteProximity/actions/workflows/main.yaml)
-
 # DevNoteProximity
```

Comment:
Please return this: 
[![Angular GitHub CI/CD](https://github.com/Nikemaks/DevNoteProximity/actions/workflows/main.yaml/badge.svg)](https://github.com/Nikemaks/DevNoteProximity/actions/workflows/main.yaml)